From a12f838a6c0767b58a06a9d1fb1fc35c99490383 Mon Sep 17 00:00:00 2001
From: Vivek R Jadhav <jadhavvivek@google.com>
Date: Fri, 22 Aug 2025 12:37:05 +0000
Subject: [PATCH] Fix build errors in Crude DNG SDK 1.7.1 upgrade

Bug: 412662901
Test: Build
(cherry picked from https://android-review.git.corp.google.com/q/commit:a74f9024a4c1505d3a7869b6e32ad9755c988a01)
Merged-In: Ib2ea03a98330c0d78b8e20a7a4dca5c82f40635f

Change-Id: I1d5adb497bf0967541215e30c3bf64e5b5bd8a73
---
 Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Android.bp b/Android.bp
index 02e2d11..334892f 100644
--- a/Android.bp
+++ b/Android.bp
@@ -288,6 +288,7 @@ cc_library {
             enabled: true,
         },
     },
+    sdk_version: "current",
 }
 
 // Build static library against the NDK
-- 
2.52.0.rc1.455.g30608eb744-goog

